DeskCheck/DeskCheck/Controllers/DesksController.cs

```diff
@@ -35,21 +30,35 @@ public IEnumerable<Desk> GetDesks()
                 registered = rng.Next(0, 2) == 1
             })
             .ToArray();
-
         }
 
         // GET: desk/getAll
-        [HttpGet("{id}")]
-        public async Task<ActionResult<Desk>> GetDesk(int id)
+        [HttpGet("getAll")]
+        public IEnumerable<Desk> GetDesks()
         {
-            var desk = await _context.Desks.FindAsync(id);
+            return desks;
+        }
 
-            if (desk == null)
+        // GET: desk/getDesk
+        [HttpGet("getDesk/{id}")]
+        public Desk GetDesk(int id)
+        {
+
+            foreach(var desk in desks){
+                if(desk.deskID == id)
+                {
+                    return desk;
+                }
+            }
+
+            return null;
+
+            /*if (desk == null)
             {
                 return NotFound();
             }
 
-            return desk;
+            return desk;*/

```

Comment:
@Jhadkin Maybe remove these commented out lines if they are not needed? Unless you think that they might be useful in the future